src/components/Alumni/AlumniCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import EmptyProfile from "@site/static/img/alumni/empty_profile.png"
+
+import Microsoft from "@site/static/img/alumni/logos/microsoft.png"
+
+function AlumniCard({name, title, company, faculty, img}) {
+    const imgOnError = ({currentTarget}) => {
+        currentTarget.onError = null
+        currentTarget.src = EmptyProfile
+    }
+    const logoLib = {
+        "Microsoft": Microsoft,
+    }
+    return (
+        <div className="w-[23.76669rem] h-[13.03331rem] rounded-[6.25rem] bg-[#2B3950]
+            flex flex-row py-[2.06rem] content-center
+            font-content">
+            <img src={img} onError={imgOnError} className="w-[8rem] h-[8rem] rounded-full mt-[0.56rem] ml-[1.69rem] mr-[1.8rem]
```

Comment:
just curious... what's the onError usecase for? is it for e.g. corrupted images that can't load?

Reply:
Yeap, you're right.